WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingAddresses/WooShippingEditAddressViewModel.swift

```diff
@@ -12,19 +23,27 @@ final class WooShippingEditAddressViewModel: ObservableObject, Identifiable {
     @Published var postalCode: String
     @Published var email: String
     @Published var phone: String
-    @Published var saveAsDefault: Bool
+    @Published var isDefault: Bool
```

Comment:
Minor thing, can we make this property more meaningful? What is default here, the address? in that case, can we name it as `isDefaultAddress`?

Reply:
Yep, good point; updated both the property and the init parameter in babdfa1.